src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -271,6 +273,47 @@ def _path(ws, path):
     ]
 
 
+def _redash_permissions_mapping():
+    def _simple(_, object_id):
+        return object_id
+
+    return [
+        (
+            "query_policy",
```

Comment:
Something completely slipped my mind: what is query policy?... did you mean just query?.. 

P.s. we have cluster policy and cluster

Reply:
corrected